traffic_portal/app/src/common/modules/form/user/form.user.tpl.html

```diff
@@ -105,8 +105,19 @@
                     <textarea name="publicSshKey" rows="5" class="form-control" ng-model="user.publicSshKey"></textarea>
                 </div>
             </div>
+            <div class="form-group" ng-if="user.newUser" ng-class="{'has-error': hasError(userForm.registrationSent), 'has-feedback': hasError(userForm.registrationSent)}">
+                <label class="control-label col-md-2 col-sm-2 col-xs-12">Registration Sent</label>
+                <div class="col-md-10 col-sm-10 col-xs-12">
+                    <input name="registrationSent" type="text" class="form-control" ng-model="user.registrationSent" ng-maxlength="128" required ng-disabled="!settings.isNew">
+                    <small class="input-error" ng-show="hasPropertyError(userForm.registrationSent, 'required')">Required</small>
```

Comment:
i don't think this should be required. it seems like we have some data that has newUser=true and registrationSent=null so you end up with this:

![image](https://user-images.githubusercontent.com/251272/106055574-c756e680-60aa-11eb-8f0e-0f1c65a0f40b.png)

Reply:
actually, let's do this. can you remove `ng-if="user.newUser"`. I guess I don't really see the need for it. and also make that field not required but leave it disabled.

Reply:
actually, can you change it to `ng-if="!settings.isNew"` - let's always show that field for existing users.

Reply:
also change `ng-disabled="!settings.isNew"` to `disabled` - there's no reason why this field would ever be enabled.

Reply:
Ok, and I can add if condition to show Registration Sent only if there is value?

Reply:
no, show if no value as well. it's fine if it's blank.

Reply:
just change `ng-if="user.newUser"` to `ng-if="!settings.isNew"`

Reply:
ok!